test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -398,7 +428,11 @@ class HttpStream : public LinkedObject<HttpStream> {
         ENVOY_LOG_MISC(debug, "Setting dispatching action  on {} in state {} {}", stream_index_,
                        static_cast<int>(request_.stream_state_),
                        static_cast<int>(response_.stream_state_));
-        auto request_action = stream_action.dispatching_action().directional_action_selector_case();
+        // auto request_action =
+        // stream_action.dispatching_action().directional_action_selector_case();
+        //  I think we should have it be on the reuqest action taken, not on
+        //  what the dispatching_action e.g. what to do in response is...
+        auto request_action = stream_action.request().directional_action_selector_case();
```

Comment:
I disaggree with using the `request()` here. See comment in [codec_impl_fuzz.proto](https://github.com/envoyproxy/envoy/blob/44634cc608f11bed6852c67f40902cea0d5c9554/test/common/http/codec_impl_fuzz.proto#L43-L47). The "directional action" is meant to be some kind of hybrid and with your change, it is not taken into account anymore.